Makefile

```diff
@@ -5,7 +5,7 @@ TARGET := kube-vip
 .DEFAULT_GOAL := $(TARGET)
 
 # These will be provided to the target
-VERSION := v0.9.1
+VERSION := v0.9.2
```

Comment:
Can we bump the makefile in a separate PR, I've some additional changes I'll need to add before we cut a new release. (mainly a cosmetic thing around kernel modules)

Reply:
Done